app/models/court_application_defendant_summary.rb

```diff
@@ -0,0 +1,61 @@
+# frozen_string_literal: true
+
+class CourtApplicationDefendantSummary
+  include ActiveModel::Model
+
+  attr_reader :defendant
+  attr_accessor :defendant_id
```

Comment:
do we need a setter here?
if not attr_reader is enough:

`attr_reader :defendant, :defendant_id

Reply:
We set defendant_id from other classes